src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -55,10 +55,30 @@
   <UsingTask AssemblyFile="$(XPlatSourceBuildTasksAssembly)" TaskName="WritePackageUsageData" />
   <UsingTask AssemblyFile="$(XPlatSourceBuildTasksAssembly)" TaskName="WriteUsageReports" />
 
-  <!-- Update NuGet feeds in the repo -->
+  <Target Name="CopyNuGetConfig"
+          BeforeTargets="
+            UpdateNuGetConfig;
+            Build"
+          Condition="'$(NuGetConfigFile)' != ''"
+          Inputs="$(MSBuildProjectFullPath)"
+          Outputs="$(BaseIntermediateOutputPath)CopyNuGetConfig.complete">
+
+    <MakeDir Directories="$([System.IO.Path]::GetDirectoryName('$(NuGetConfigFile)'))" />
+
+    <Copy
+      SourceFiles="$(OriginalNuGetConfigFile)"
+      DestinationFiles="$(NuGetConfigFile)" />
```

Comment:
Add the `CopiedFiles` output to TouchedFiles:

`<Output TaskParameter="CopiedFiles" ItemName="FileWrites" />`

Reply:
I think this is immaterial now that we don't create the semaphore.

Reply:
FileWrites isn't related to semaphore files. You still want to set this here. This makes sure that incremental builds for this project work correctly.

Reply:
Yes, this isn't super important.